### ISSUE TITLE
Module Build Failed: Cannot read property 'originalPositionFor' of undefined

### ISSUE THREAD

**Issue by brettpalmberg** [tailwindlabs/headlessui#1085]
**What package within Headless UI are you using?**

@headlessui/react@latest

**What version of that package are you using?**

@headlessui/react@latest (^1.4.3)

_Note: Tried @headlessui/react@insiders (0.0.0-insiders.6fc28c6), and the bug was not present_

**What browser are you using?**

Chrome

**Reproduction URL**

https://github.com/brettpalmberg/headlessui-bug-demo-01

This is a clean-slate repository, bootstrapped with `create-react-app`.  The app tries to render the headlessui Menu (Dropdown) example component here: https://headlessui.dev/react/menu, resulting in the error below.

**Describe your issue**

Error message from react app is copied below:

```
Compiled with problems:X

ERROR in ./node_modules/@headlessui/react/dist/index.esm.js

Module build failed (from ./node_modules/babel-loader/lib/index.js):
TypeError: /Users/rdcrlbtp/Documents/projects/cumulus-ui-too/node_modules/@headlessui/react/dist/index.esm.js: Cannot read property 'originalPositionFor' of undefined
    at SourceMapTree.originalPositionFor (/Users/rdcrlbtp/Documents/projects/cumulus-ui-too/node_modules/@ampproject/remapping/dist/remapping.umd.js:159:27)
    at trace (/Users/rdcrlbtp/Documents/projects/cumulus-ui-too/node_modules/@ampproject/remapping/dist/remapping.umd.js:102:37)
    at EncodedSourceMapImpl.map (/Users/rdcrlbtp/Documents/projects/cumulus-ui-too/node_modules/@jridgewell/trace-mapping/dist/trace-mapping.umd.js:347:36)
    at TraceMap.map (/Users/rdcrlbtp/Documents/projects/cumulus-ui-too/node_modules/@jridgewell/trace-mapping/dist/trace-mapping.umd.js:430:31)
    at SourceMapTree.traceMappings (/Users/rdcrlbtp/Documents/projects/cumulus-ui-too/node_modules/@ampproject/remapping/dist/remapping.umd.js:85:34)
    at remapping (/Users/rdcrlbtp/Documents/projects/cumulus-ui-too/node_modules/@ampproject/remapping/dist/remapping.umd.js:258:36)
    at mergeSourceMap (/Users/rdcrlbtp/Documents/projects/cumulus-ui-too/node_modules/@babel/core/lib/transformation/file/merge-map.js:19:30)
    at generateCode (/Users/rdcrlbtp/Documents/projects/cumulus-ui-too/node_modules/@babel/core/lib/transformation/file/generate.js:72:39)
    at run (/Users/rdcrlbtp/Documents/projects/cumulus-ui-too/node_modules/@babel/core/lib/transformation/index.js:55:33)
    at run.next (<anonymous>)```

**Comment by cinthia3301andrad** [comment]
I had the same problem, I tried everything. Oh I found out that

There was a problem inside the @jridgewell/trace-mapping package.

The problem has now been fixed and the new version v0.2.3 has been published.

All you need to do is run npm update or yarn update

**Comment by brettpalmberg** [comment]
Excellent - this appears to be working in my project now too.  Thank you very much for sharing @cinthia3301andrad.

Please feel free to close this issue now & have a good day all.
cc @adamwathan @reinink @davidluhr
